src/components/date-picker/date-picker.tsx

```diff
@@ -202,13 +234,16 @@ export class DatePicker implements LocalizedComponent {
 
   @State() globalAttributes = {};
```

Comment:
Sidebar: this doesn't seem to be used at all.

Reply:
This is for accessing the global `lang` attribute.

Reply:
that's wired-up in the new localization interface.

Reply:
`LocalizedComponent` does not rely on this.